nx.json

```diff
@@ -29,7 +29,7 @@
         "currentVersionResolverMetadata": {
           "tag": "latest"
         },
-        "preid": "next",
```

Comment:
This is used to put `rc` or ` nightly` in the version string. Why was it set to `next`?

Reply:
Short answer: I thought `next` was the string to create rc's.